src/views/Header.vue

```diff
@@ -0,0 +1,105 @@
+
+
+<script>
```

Comment:
this component is bad because you put too many logic there.

Header is a component that should be visible across all pages so it should be in the components folder